Letterboxd

dredvard opened the pull request:
Added the ability to do letterboxd and corrected the issue where it will push the wrong movie (e.g Psycho -1960 vs 1998) by using release date.  This only does it for letterboxd as this method should be i modified for other list types.

I've included plugins from Auto-Collections (which does the same pulling) - and does it for a lot more types.  These 2 projects work hand in hand.  One pulls it and puts on Jellyseer/Overseer - the other then adds it as a collection to Jellyfin.  The other major difference is that pulls its initial data from a config file.  Config concept is easier to setup for an initial config, but your terminal is very handy for troubleshooting and testing.   I could see a config file where you specify -  Jellyfin Collections or Overseerr update for any given lists.

There are a few ways to implement this based on how you want to do it so I only made letterboxd work and demonstrated how to do it.  You can implemenet the same methodoly for all .    I've also based this on refactored as this is file organization I think works best with this.

Woahai321 commented:
hello @dredvard 

First of all, my apologies i did not see this pull request and re started work on this project after the holidays coming to and end etc.

This project is inadvertently based on the autocollections project you mention, i used this one time on my jellyfin deployment and thought why cant this go to seerr instead and this is the lovechild of learning and fixing a problem for me. 
I understand there is a big hole in the jellyfin pipeline this has the potential to fix really well and aim for this to have a webui configuration and be a proper tool in due course.

Since your PR we have changed how we are scraping the respective list sites to use browser automation as opposed to web scraping in an attempt to 1) make the data collection more functional and ready for more features in the future & 2) want to learn more about browser automation and this was the perfect place to do so.

As a result of the changes, this PR is no longer applicable even though it is really well formatted and exactly what i am going for.

Would it be possible for you to creeate a new PR refactoring this release based on v0.5.2 and I can merge this before making any other changes.

This would be highly appreciated and would go a long way to making this project the beast I am aiming for!

Please let me know if this is something you are willing to help with @dredvard

Woahai321 commented:
closing due to letterboxd integration available and will refactor myself in due course, thank you fot the thought though!

dredvard commented:
Had to get back to work yesterday so I haven't had a chance to look at your
modifications.  I see letterboxd now in Main (5.4).  Thanks.  Havent worked
with Selinium so I'm completely unfamiliar with that - so I'm glad you
refactored it.  I see you've also fixed the year match logic.  FYI Poetry
and requirements.txt were out of sync for the version I worked on).  Maybe
some sort of automation would make sure both stay in sync if you work with
1 vs the other (or just specifiy the one you are using) or fail the push if
one isn't.

So the current state is that you just have implemented the checks for the
letterbox (  Does this logic actually work?: `list_id in ['top',
'boxoffice', 'moviemeter', 'tvmeter']:   - do you mean?  any(item in
list_id for item in   ['top', 'boxoffice', 'moviemeter', 'tvmeter']  ) `

One other issue I am having is that it doesn't repetively schedule
anything.  If you leave this running in a vm, it will execute the first
wait - but then wakes after sleep and requires user input before executing
anything  further.




On Sat, Jan 4, 2025 at 2:50 AM WoahAI ***@***.***> wrote:

> closing due to letterboxd integration available and will refactor myself
> in due course, thank you fot the thought though!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Woahai321/list-sync/pull/14#issuecomment-2570593691>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAMCFZZEJWV7O7HDNWRVAFL2I6HFBAVCNFSM6AAAAABUIYDCRCVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDKNZQGU4TGNRZGE>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
